RealmSwift-swift2.0/Tests/MigrationTests.swift

```diff
@@ -146,6 +149,7 @@ class MigrationTests: TestCase {
         }
     }
 
+    // swiftlint:disable function_body_length
```

Comment:
Is there a good reason not to just disable this for the entirety of test files where this is an issue? I don't think that toggling it on and off in such code buys us anything unless we intend to fix these issues at some point.